pkg/cmd/connector/connector_type/describe/describe.go

```diff
@@ -0,0 +1,92 @@
+package describe
+
+import (
+	"github.com/redhat-developer/app-services-cli/pkg/cmd/connector/connectorcmdutil"
+	"github.com/redhat-developer/app-services-cli/pkg/core/cmdutil/flagutil"
+	"github.com/redhat-developer/app-services-cli/pkg/core/ioutil/dump"
+	"github.com/redhat-developer/app-services-cli/pkg/core/localize"
+
+	"github.com/redhat-developer/app-services-cli/pkg/shared/connection"
+	"github.com/redhat-developer/app-services-cli/pkg/shared/factory"
+	connectorerror "github.com/redhat-developer/app-services-sdk-go/connectormgmt/apiv1/error"
+
+	"github.com/spf13/cobra"
+)
+
+type options struct {
+	type_id      string
+	outputFormat string
+
+	f *factory.Factory
+}
+
+// NewDescribeCommand creates a new command to view a connector type
+func NewDescribeCommand(f *factory.Factory) *cobra.Command {
+	opts := &options{
+		f: f,
+	}
+
+	cmd := &cobra.Command{
+		Use:     "describe",
+		Short:   f.Localizer.MustLocalize("connector.type.describe.cmd.shortDescription"),
+		Long:    f.Localizer.MustLocalize("connector.type.describe.cmd.longDescription"),
+		Example: f.Localizer.MustLocalize("connector.type.describe.cmd.example"),
+		Args:    cobra.NoArgs,
+		RunE: func(cmd *cobra.Command, args []string) error {
+
+			validOutputFormats := flagutil.ValidOutputFormats
+			if opts.outputFormat != "" && !flagutil.IsValidInput(opts.outputFormat, validOutputFormats...) {
+				return flagutil.InvalidValueError("output", opts.outputFormat, validOutputFormats...)
+			}
+
+			return runDescribe(opts)
+		},
+	}
+	flags := connectorcmdutil.NewFlagSet(cmd, f)
+	flags.StringVar(&opts.type_id, "type", "", f.Localizer.MustLocalize("connector.type.describe.flag.id"))
+	flags.AddOutput(&opts.outputFormat)
+
+	_ = cmd.MarkFlagRequired("type")
```

Comment:
No interactive mode? I guess picking up from 200 types is not great? Honest question. Not an requirement